bundles/ui/org.eclipse.smarthome.ui/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImpl.java

```diff
@@ -173,6 +174,9 @@ public Widget getWidget(String itemName) {
         for (ItemUIProvider provider : itemUIProviders) {
             Widget currentWidget = provider.getWidget(itemName);
             if (currentWidget != null) {
+                if (currentWidget instanceof Default) {
```

Comment:
Okay, but then you can remove the <code>if (currentWidget instanceof Default</code> because this is already handled corretly in the resolveDefault function. Isn't it?